### PR TITLE
feat(plugin): enable Clamp to ground property

### PR DESCRIPTION
**Pull request by nina992**
# Overview 
We need to enable Clamp to ground property for polygon, marker and models.

### CODE REVIEW

File: plugin/src/sidebar.ts [hunk 1]
```diff
@@ -584,12 +584,14 @@ function createLayer(dataset: DataCatalogItem, overrides?: any) {
             style: "point",
             pointSize: 10,
             pointColor: "white",
+            clampToGround: true,
```
Comment (keiya01) [inline plugin/src/sidebar.ts:587]: About marker, we need to use heightReference.

```suggestion
            heightReference: "clamp",
```

**Review by keiya01 — COMMENTED** [review]
Also I think we need to set clampToGround setting to CZML in [this line](https://github.com/eukarya-inc/reearth-plateauview/blob/3fcfc802514590ba4d8c6f2300e79578cff64b58/plugin/src/sidebar.ts#L607) like the following.

```js
{
  resource: {},
  marker: { heightReference: "clamp" },
  polyline: { clampToGround: true },
  polygon: { clampToGround: true },
}
```

File: plugin/src/sidebar.ts (hunk 1 above)
Comment (nina992) [reply plugin/src/sidebar.ts:587]: Done

**Review by keiya01 — APPROVED** [review]

File: plugin/src/sidebar.ts (hunk 1 above)
Comment (nina992) [reply plugin/src/sidebar.ts]: 83e66e860c7ad90109b6c1185f6cdcbb085c5d10

**Pull request merged**